src/components/splash-screen/styles.ts

```diff
@@ -15,6 +15,6 @@ export default (theme: ITheme) =>
         lottie: {
             ...StyleSheet.absoluteFillObject,
             height: '100%',
-            top: -150
+            top: -BASE_DIMENSION * 10
```

Comment:
This dimension needs to be exact? Is needed so the animation exists screen to the top?
If not, is better to specify larger dimension in percents.